starknet_py/net/l1/messages.py

```diff
@@ -0,0 +1,95 @@
+from dataclasses import dataclass
+from functools import reduce
+from typing import List
+from eth_utils import keccak
+
+from web3.providers import BaseProvider as L1Provider
+from starknet_py.net.l1.contracts import StarknetL1Contract
+from starknet_py.net.l1.networks import L1Network
+
+
+def encode_packed(*args: List[int]) -> bytes:
+    return reduce(
+        lambda acc, x: acc + x,
+        map(lambda x: x.to_bytes(32, byteorder="big", signed=False), args),
```

Comment:
`[x.to_bytes(32, byteorder="big", signed=False)  for x in args]` is shorter and simpler.